spatialpy/Mesh.py

```diff
@@ -28,14 +29,14 @@ def __init__(self, numpoints, xlim, ylim, zlim, rho0=1.0, c0=10, P0=10, gravity=
         self.ylim = ylim
         self.zlim = zlim
 
-    def add_point(self, x, vol, mass, type, nu, fixed):
-        self.vol = numpy.append(self.vol,vol)
-        self.mass = numpy.append(self.mass,mass)
-        self.type = numpy.append(self.type,type)
-        self.nu = numpy.append(self.nu,nu)
-        self.fixed = numpy.append(self.fixed,fixed)
+        def add_point(self, x, vol, mass, type, nu, fixed):	
+            self.vol = numpy.append(self.vol,vol)	
+            self.mass = numpy.append(self.mass,mass)	
+            self.type = numpy.append(self.type,type)	
+            self.nu = numpy.append(self.nu,nu)	
+            self.fixed = numpy.append(self.fixed,fixed)	
 
-        self.vertices = numpy.append(self.vertices, [x], axis=0)
+            self.vertices = numpy.append(self.vertices, [x], axis=0)
```

Comment:
add_point() is indented one level too far.

Reply:
oh good move!  I don't know why it marked it as a delete in the first place...